src/Azure.Functions.Cli/StaticResources/Dockerfile.dotnet9Isolated

```diff
@@ -0,0 +1,14 @@
+FROM mcr.microsoft.com/dotnet/sdk:9.0 AS installer-env
+
+COPY . /src/dotnet-function-app
+RUN cd /src/dotnet-function-app && \
+mkdir -p /home/site/wwwroot && \
+dotnet publish *.csproj --output /home/site/wwwroot
+
+# To enable ssh & remote debugging on app service change the base image to the one below
+# FROM mcr.microsoft.com/azure-functions/dotnet-isolated:4-dotnet-isolated9.0-appservice
+FROM mcr.microsoft.com/azure-functions/dotnet-isolated:4-dotnet-isolated9.0
```

Comment:
@aishwaryabh Do we know when this image will be available?

Reply:
Just noting that Image availability does not impact our ability to merge and release these changes.

Reply:
image is now available